src/main/webapp/enroll.html

```diff
@@ -14,55 +20,80 @@
         <h1>Product Release</h1>      
     </div> 
 
-    <div class="parent-container d-flex">
-        <div class="container">
-            <div class="row">
-                <div class="col">
-                    <div class="container mt-3">
-                        <h2>SDKs</h2>
-                        <form action="/action_page.php">
-                            <select name="sdks" class="custom-select mb-3">
-                            <option selected>Select SDK</option>
-                            <option value="firebase_common">Firebase Common</option>
-                            <option value="firebase_messaging">Firebase Messaging</option>
-                            <option value="firebase_idd">Firebase IID</option>
-                            <option value="firebase_storage_api">Firebase Storage API</option>
-                            <option value="firebase_ads">Firebase Ads</option>
-                            <option value="tagmanager">Tagmanager</option>
-                            </select>
-                            <button type="submit" class="btn btn-primary">Submit</button>
-                        </form>
-                    </div>
+    <div class="container">
+        <div class="stepwizard">
+            <div class="stepwizard-row setup-panel">
+                <div class="stepwizard-step col-xs-3"> 
+                    <a href="#step-1" type="button" class="btn btn-success btn-circle">1</a>
+                    <p><small>Select SDK</small></p>
+                </div>
+                <div class="stepwizard-step col-xs-3"> 
+                    <a href="#step-2" type="button" class="btn btn-default btn-circle" disabled="disabled">2</a>
+                    <p><small>Current Version</small></p>
+                </div>
+                <div class="stepwizard-step col-xs-3"> 
+                    <a href="#step-3" type="button" class="btn btn-default btn-circle" disabled="disabled">3</a>
+                    <p><small>Version Being Released</small></p>
+                </div>
+                <div class="stepwizard-step col-xs-3"> 
+                    <a href="#step-4" type="button" class="btn btn-default btn-circle" disabled="disabled">4</a>
+                    <p><small>Link to Cl Notes</small></p>
```

Comment:
I think the name of the enrollment step should be more generic than just repeating what input is being asked for in the step.
You can change this to "Release details", so we can add other fields later on the same page.

You could also avoid naming the steps completely. That might be easier and cleaner.